koku/koku/notifications.py

```diff
@@ -96,10 +96,10 @@ def send_notification(self, msg):
         """
         producer = get_producer()
         producer.produce(Config.NOTIFICATION_TOPIC, value=msg, callback=delivery_callback)
-        # Wait up to 1 second for events. Callbacks will be invoked during
+        # Wait up to 3 seconds for events. Callbacks will be invoked during
         # this method call if the message is acknowledged.
         # `flush` makes this process synchronous compared to async with `poll`
-        producer.flush(1)
+        producer.flush(3)
```

Comment:
We can just make this `.poll(0)`. We don't need to block to confirm the message was delivered. We don't even retry to send the message if it fails, so there is no point in waiting for confirmation.

Reply:
Switched again to using `.poll(0)` in the places we send a message.